shared/pages/import/js/curtain_panel.js

```diff
@@ -0,0 +1,291 @@
+'use strict';
+/* global LazyLoader */
+/* exported Curtain */
+
+/*
+ * The name it's quite confusing, but we are going to work in several
```

Comment:
Great, thanks for the doc, let's add just at the begining `XXX:` to represent that is something we want to change